internal/api/api.go

```diff
@@ -227,12 +252,35 @@ func (api CertAPIClient) GetOperatorBundleIDByPackageName(org, name, vsersion st
 	} else {
 		url = fmt.Sprintf("%s/bundles?page_size=1&filter=organization==%s;csv_name==%s", apiOperatorCatalogExternalBaseEndPoint, org, name)
 	}
+
 	responseData, err := api.getRequest(url)
 	if err == nil {
 		imageID, err = api.getIDFromResponse(responseData)
 	}
+
 	return imageID, err
 }
+func (api CertAPIClient) GetYamlFile() (ChartStruct, error) {
+	url := ("https://charts.openshift.io/index.yaml")
+	responseData, err := api.getRequest(url)
+	var body interface{}
+	var charts ChartStruct
+	if errorr := yaml.Unmarshal(responseData, &body); errorr != nil && err != nil {
+		panic(errorr)
+	}
+
+	body = convertYamlToJSON(body)
+
+	if b, eror := json.Marshal(body); eror != nil {
+		panic(eror)
```

Comment:
if you panic here the program will stop. 
you should log an error, mark the test as failed.